app/static/src/app/components/WodinPanels.vue

```diff
@@ -128,31 +238,35 @@ export default defineComponent({
         cursor: pointer;
         width: 10rem;
         color: $dark-grey;
+        transform: rotate(90deg);
     }
 
-    .view-left {
-        transform: rotate(90deg) translate(6rem, 2.7rem);
+    .view-left-expand {
+        position: absolute;
+        right: -2.7rem;
+        top: 7rem
+    }
+
+    .view-left-collapsed {
+        position: absolute;
+        right: -2.7rem;
+        top: 7rem
     }
```

Comment:
Looks like these two classes are the same, could just have the content once?

Reply:
done, in fact i dont even need these classes, can just shove this into view-left, simplifies the code a bit!